Azure/azure-cli
# [AppService] Fix #13888 : Add support for Static WebApps: get, list, create commands

**Description**  
Add new commands for new public preview resource type: Static WebApps. Fixes #13888

**Testing Guide**  

TBD


---

This checklist is used to make sure that common guidelines for a pull request are followed.
